lib/handler.js

```diff
@@ -30,5 +30,10 @@ module.exports = (viewConfig) => {
       }
       throw boom.boomify(err);
     }
+    const viewOutput = h.view(viewConfig.view, result);
+    if (request.query.json === '1') {
```

Comment:
Not right, json wouldn't run through view

Reply:
Also need to check config var.

Reply:
We should double check that the config vars are actually still used. Looks like some left overs